src/sentry/static/sentry/app/views/organizationIntegrations/integrationPluginRow.tsx

```diff
@@ -0,0 +1,111 @@
+import {withTheme} from 'emotion-theming';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import Link from 'app/components/links/link';
+import {PanelItem} from 'app/components/panels';
+import {t} from 'app/locale';
+import CircleIndicator from 'app/components/circleIndicator';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {PluginWithProjectList} from 'app/types';
+
+type Props = {
+  plugin: PluginWithProjectList;
+  legacy: boolean;
+};
+
+export default class PluginRow extends React.Component<Props> {
+  static propTypes = {
+    plugin: PropTypes.object.isRequired,
+  };
+
+  static contextTypes = {
+    organization: SentryTypes.Organization,
+  };
+
+  get isEnabled() {
+    return this.props.plugin.projectList.length > 0;
+  }
+
+  render() {
+    const {plugin, legacy} = this.props;
+    const {
+      organization: {slug},
+    } = this.context;
+    return (
+      <PanelItem p={0} flexDirection="column" data-test-id={plugin.id}>
+        <Flex style={{alignItems: 'center', padding: '16px'}}>
+          <PluginIcon size={36} pluginId={plugin.id} />
+          <div style={{flex: '1', padding: '0 16px'}}>
```

Comment:
I think it's better to stick to either Emotion or inline-styles. Personally I'd go with emotion.

Reply:
Also make sure to use the `app/styles/space.tsx` file for things like padding and margin, instead of hard-coding `px` values

Reply:
We should definitely use styled-components to do the styling.